build

```diff
@@ -14,7 +14,9 @@ export GOPATH=${PWD}/gopath
 eval $(go env)
 
 echo "Building rkt (stage0)..."
-go build -o $GOBIN/rkt ${REPO_PATH}/rkt
+eval go build -o ${GOBIN}/rkt \
+	${RKT_STAGE1_IMAGE:+-ldflags \'-X main.defaultStage1Image "${RKT_STAGE1_IMAGE}"\'} \
```

Comment:
there is probably a better way of doing this than using eval but my bash fu is terrible, so improvements gladly welcomed.